src/test/java/jpos/config/simple/xml/JavaxRegPopulatorTestCase.java

```
@@ -287,6 +289,108 @@ public void testGetLastLoadException()
+
+            assertTrue( "Expected an empty set of entries...", JUnitUtility.isIdentical( entries, Collections.enumeration(v1) ) );
+            assertTrue( "Expected an empty set of entries...", JUnitUtility.isEquals( entries, Collections.enumeration(v1) ) );
+
+            //Add some entries and save and load
+            JposEntry entry1 = createJposEntry( "entry1", "com.xyz.jpos.XyzJposServiceInstanceFactory",
+                    "com.xyz.jpos.LineDisplayService", "Xyz, Corp.",
+                    "http://www.javapos.com", "LineDisplay", "1.4a",
+                    "Virtual LineDisplay JavaPOS Service",
+                    "Example virtual LineDisplay JavaPOS Service from virtual Xyz Corporation",
+                    "http://www.javapos.com" );
+
+
+            v1.clear();
+            v1.add( entry1 );
+
+            javaxRegPopulator.save( Collections.enumeration(v1) );
+            javaxRegPopulator.load( );
+
+            assertNull(javaxRegPopulator.getLastLoadException());
+
+            @SuppressWarnings("unchecked")
+            Enumeration<JposEntry> entries2 = javaxRegPopulator.getEntries();
+
+            assertTrue( "Expected 1 entry...", JUnitUtility.isEquals( entries2, Collections.enumeration(v1) ) );
+
+            //Remove entries save and load reg
+            v1.remove( entry1 );
+
+            javaxRegPopulator.save( Collections.enumeration(v1) );
+            javaxRegPopulator.load( );
+
+            assertNull(javaxRegPopulator.getLastLoadException());
+
+            @SuppressWarnings("unchecked")
+            Enumeration<JposEntry> entries3 = javaxRegPopulator.getEntries();
+
+            assertTrue( "Expected 1 entries...", JUnitUtility.isEquals( entries3, Collections.enumeration(v1) ) );
+        }
+        catch( Exception e )
+        {
+            assertTrue( "Got unexpected Exception from XercesRegPopulator.save method with message = " + e.getMessage(), true );
```

Why not let it fail with 
```java
fail(e.getMessage());
```

---

OK, I will modify it now and update the message, and still talks about `XercesRegPopulator`.  Copy-paste error.

---

Done. Made exception the same as in some other case.